topiary/rna/cufflinks.py

```diff
@@ -19,6 +19,21 @@
 import pandas as pd
 import numpy as np
 
+from .common import infer_separator, check_required_columns
+
+
+def parse_locus_column(loci):
+    # capture all characters after 'chr' but before ':'
```

Comment:
Is this comment still accurate if you're not assuming it starts with `chr` anymore?